addons/rookupgrade/10to14/install.sh

```diff
@@ -93,9 +93,16 @@ function rookupgrade_10to14_upgrade() {
 
         log "Waiting for Rook 1.1.9 to rollout throughout the cluster, this may take some time"
         if ! "$DIR"/bin/kurl rook wait-for-rook-version "v1.1.9" --timeout=1200 ; then
-            logWarn "Detected multiple Rook versions"
+            logWarn "Timeout waiting for Rook version 1.1.9 rolled out"
+            logStep "Checking Rook versions and replicas"
             kubectl -n rook-ceph get deployment -l rook_cluster=rook-ceph -o jsonpath='{range .items[*]}{.metadata.name}{"  \treq/upd/avl: "}{.spec.replicas}{"/"}{.status.updatedReplicas}{"/"}{.status.readyReplicas}{"  \trook-version="}{.metadata.labels.rook-version}{"\n"}{end}'
-            bail "Failed to verify the Rook upgrade, multiple Rook versions detected"
```

Comment:
this is a functional change rather than only messaging. this should bail as it did before.